doc/rst/meta/modules/packages/Sort.rst

```diff
@@ -0,0 +1,11 @@
+.. TODO: remove this stub file
```

Comment:
It might be helpful to put a timeline on this. Maybe we can remove this after a normal deprecation cycle? Or a different timeline if that makes more sense.
```suggestion
.. TODO: remove this stub file after Chapel 2.3
```